bundles/org.openhab.binding.folderwatcher/README.md

```diff
@@ -1,54 +1,66 @@
+| diffHours         | Time stamp difference, h | How many hours back to analyze      | yes      | 24            |
 
 The `localfolder` thing has the following configuration options:
 
-| Parameter   | Name         | Description                                                                                                            | Required | Default value |
-|-------------|--------------|------------------------------------------------------------------------------------------------------------------------|----------|---------------|
-| localDir    | Local Directory | Local directory to be watched                                                                                       | yes      | n/a           |
-| listHiddenLocal | List Hidden | Allow listing of hidden files                                                                                       | yes      | No            |
-| pollIntervalLocal | Polling interval, s | Interval for polling folder changes                                                                       | yes      | 60            |
-| listRecursiveLocal | List Sub Folders | Allow listing of sub folders                                                                                | yes      | No            |
-
+| Parameter          | Name                | Description                         | Required | Default value |
+|--------------------|---------------------|-------------------------------------|----------|---------------|
+| localDir           | Local Directory     | Local directory to be watched       | yes      | n/a           |
+| listHiddenLocal    | List Hidden         | Allow listing of hidden files       | yes      | No            |
+| pollIntervalLocal  | Polling interval, s | Interval for polling folder changes | yes      | 60            |
+| listRecursiveLocal | List Sub Folders    | Allow listing of sub folders        | yes      | No            |
+
+The `s3bucket` thing has the following configuration options:
+
+| Parameter      | Name                 | Description                                    | Required | Default value |
+|----------------|----------------------|------------------------------------------------|----------|---------------|
+| s3BucketName   | S3 bucket name       | Name of the S3 bucket to be watched            | yes      | n/a           |
+| s3Path         | S3 path              | S3 path (folder) to be monitored               | no       | n/a           |
+| pollIntervalS3 | Polling Interval     | Interval for polling S3 bucket changes, sec    | yes      | 60            |
+| awsKey         | AWS access key       | AWS access key                                 | no       | n/a           |
+| awsSecret      | AWS secret           | AWS secret                                     | no       | n/a           |
+| awsRegion      | AWS Region           | AWS Region of S3 bucket                        | yes      | ""            |
```

Comment:
For consistency:
```suggestion
| awsRegion      | AWS region           | AWS region of S3 bucket                        | yes      | ""            |
```

or perhaps the names should be consistent with the config parameter labels (that were just updated)?